client/tasks/fills/woocommerce-payments.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { registerPlugin } from '@wordpress/plugins';
+import { WooOnboardingTaskListItem } from '@woocommerce/onboarding';
+import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+import { useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { installActivateAndConnectWcpay } from './PaymentGatewaySuggestions/components/WCPay';
+
+const WoocommercePaymentsTaskItem = () => {
+	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
+	const { createNotice } = useDispatch( 'core/notices' );
+
+	return (
+		<WooOnboardingTaskListItem id="woocommerce-payments">
+			{ ( { defaultTaskItem, onClickActions } ) => (
+				<>
+					{ React.cloneElement( defaultTaskItem, {
+						onClick: () => {
+							onClickActions();
+
+							return new Promise( ( resolve, reject ) => {
```

Comment:
I'm not sure if it's still handled this way in the experimental task list item, but previously the `onClick` would accept a promise and automatically have a busy state until it was resolved.

If that's the case, something may be broken here.  And if not, we should probably add some indication of busy state while this is installing/activating, but also okay if we want to handle in a follow-up.

Reply:
I don't see any logic capturing a promise via the onClick handler. I _have_ noticed that some of the buttons elsewhere on the page will display a busy state whenever `installPlugins` is being requested in the plugins store, which can be triggered by clicking this task.

The busy states I've seen are triggered by buttons, and I'm not certain how this might appear when triggered by a task? @joshuatf